README.md

```diff
@@ -19,6 +19,32 @@ This repo provides a nice packaged solution to it, that plays nicely with Docker
 
 ## Synopsis
 
+### Caveats
+#### Novel Python Syntax
```

Comment:
This shouldn't be the first thing in the README -- people don't know what this repo is yet.

Reply:
Maybe move both of these to inside the `### Requirements` section of the doc?